qkit/core/instrument_base.py

```diff
@@ -733,7 +733,7 @@ def add_function(self, name, **options):
         if hasattr(f, '__doc__'):
             options['doc'] = getattr(f, '__doc__')
 
-        options['argspec'] = self.get_argspec_dict(inspect.getargspec(f))
+        options['argspec'] = self.get_argspec_dict(inspect.getfullargspec(f))
 
```

Comment:
was passiert hier?

Reply:
getargspec is deprecated und hat nicht mehr funktioniert. Getfullargspec ist der empfohlene Ersatz und sollte als 1:1-Ersetzung funktionieren.